packages/api/src/models/Business.ts

```diff
@@ -5,11 +5,11 @@ import BusinessClient, { transformBusinessClientResponse, BusinessClientResponse
 import { Nullable } from './helpers'
 
 export default interface Business {
-	id: string
+	id: number
 	name: string
 	accountId: string
 	address: Nullable<Address>
-	phoneNumber?: Nullable<PhoneNumber>
+	phoneNumber?: string
```

Comment:
For me the /business endpoint returns:

```
"phone_number": {
            "id": 67611,
            "phone_number": "18663036061"
        },
```

So I think the object is correct.

Reply:
OK, that makes sense, here however we have it mentioned as a string - should we update the docs in that case?
https://www.freshbooks.com/api/me_endpoint

Reply:
Sounds good. I'll update.